### ISSUE TITLE
Normalization

### ISSUE THREAD

**Issue by LordPontiag** [lululxvi/deepxde#2055]
Hi @lululxvi , @vl-dud 
I hope you're doing well

I'm using DeepXDE to solve Navier-Stokes equations.
This is my data section:
```
Training_POINTS = X_DNS
data = dde.data.PDE(geom,
                   pde,
                   [U_DNS,V_DNS,P_DNS,
                    bc_wall_u,bc_wall_v,
                    bc_periodic_u,bc_periodic_v,bc_periodic_p],
                    anchors= Training_POINTS,
                    num_boundary= 1500,
                    num_test = 500)
```
I want to add normalization to my code. I searched for this but I couldn't find a reliable source of how to do it. I want to know what things must be normalized; like geometry, PDEs, DNS data, boundary conditions or rest of them. And is there any way to normalize all of the inputs right before this section of code? I mean not to manipulate all code, just add a function before this to normalize them (something like output transform) and apply it to network. And after that how to denormalize to see results? which functions are available in DeepXDE for this (like sigmoid).

rest of the code is as below:

Network and training:
```
net = dde.maps.FNN([2] + [25]*5 + [3] , "tanh", "Glorot uniform")

model = dde.Model(data, net)
model.compile("adam",lr=1e-3)
losshistory, train_state = model.train(iterations = 1000, display_every = 100)

dde.optimizers.config.set_LBFGS_options(maxiter=3000)
model.compile("L-BFGS")
losshistory, train_state = model.train()
dde.saveplot(losshistory, train_state, issave=False, isplot=True)
```
results:
```
samples = geom.random_points(200000)
result = model.predict(samples)
```

**Comment by vl-dud** [comment]
Hi @LordPontiag 

You may be referring to non-dimensionalization of PDEs. If so, it means rewriting the equations in a dimensionless form (see this [example](https://en.wikipedia.org/wiki/Non-dimensionalization_and_scaling_of_the_Navier%E2%80%93Stokes_equations)). If not, and you only need input/output normalization, you can use `net.apply_feature_transform` for inputs and `net.apply_output_transform` for outputs. Which normalization to use depends on your problem and preferences.

**Comment by LordPontiag** [comment]
> Hi [@LordPontiag](https://github.com/LordPontiag)
> 
> You may be referring to non-dimensionalization of PDEs. If so, it means rewriting the equations in a dimensionless form (see this [example](https://en.wikipedia.org/wiki/Non-dimensionalization_and_scaling_of_the_Navier%E2%80%93Stokes_equations)). If not, and you only need input/output normalization, you can use `net.apply_feature_transform` for inputs and `net.apply_output_transform` for outputs. Which normalization to use depends on your problem and preferences.

Hi @vl-dud 
Thank you for your reply

No I know about non-dimensionalization, I do it using Re number in NS equations, but the point here was normalization. I used both `net.apply_feature_transform` and `net.apply_output_transform` to do a z-score normalization and it worked! Thank you for your recommendation

**Comment by echen5503** [comment]
@lululxvi seems resolved, recommend close
